src/EditorFeatures/Core/Implementation/IntelliSense/EditorCompletionImplementation/EditorAsyncCompletionServiceProvider.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Utilities;
+using EditorCompletion = Microsoft.VisualStudio.Language.Intellisense;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.EditorImplementation
+{
+    [Export(typeof(IAsyncCompletionServiceProvider))]
+    [Name("C# and Visual Basic Completion Service Provider")]
```

Comment:
Is the expectation that F#/TS/JS export this as well.  Are they already doing this?